L/LibPQ/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LibPQ"
+version = v"12.3"
+pg_version = string(version.major, '.', version.minor)
+tzdata_version = "2020a"
+
+# Collection of sources required to build LibPQ
+sources = [
+    ArchiveSource(
+        "https://ftp.postgresql.org/pub/source/v$pg_version/postgresql-$pg_version.tar.gz",
+        "708fd5b32a97577679d3c13824c633936f886a733fc55ab5a9240b615a105f50"
+    ),
+    ArchiveSource(
+        "https://data.iana.org/time-zones/releases/tzdata$tzdata_version.tar.gz",
+        "547161eca24d344e0b5f96aff6a76b454da295dc14ed4ca50c2355043fb899a2",
+    ),
+    ArchiveSource(
+        "https://data.iana.org/time-zones/releases/tzcode$tzdata_version.tar.gz",
+        "7d2af7120ee03df71fbca24031ccaf42404752e639196fe93c79a41b38a6d669",
+    ),
+]
+
+# Bash recipe for building across all platforms
+# NOTE: readline and zlib are not used by libpq
+script = raw"""
+cd $WORKSPACE/srcdir
+make CC=$BUILD_CC
+export ZIC=$WORKSPACE/srcdir/zic
+cd postgresql-*/
+./configure --prefix=$prefix --host=$target --with-includes=$prefix/include --with-libraries=$prefix/lib --without-readline --without-zlib --with-openssl
+make -C src/interfaces/libpq install
+install_license COPYRIGHT
+rm -rf ${prefix}/logs
```

Comment:
Why?  Logs are very useful for debugging.  Now they're compressed with gzip to reduce space on disk, if that's the problem.

Reply:
I honestly don't remember but the size was probably the issue, yes. I can remove that.